homeassistant/components/iskra/entity.py

```diff
@@ -0,0 +1,63 @@
+"""Base entity for Iskra devices."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, MANUFACTURER
+
+
+class IskraDevice(Entity):
```

Comment:
Shouldn't we call this an entity instead of device?

Reply:
changed